manifests/master/backup.pp

```diff
@@ -0,0 +1,19 @@
+# @summary Configure xCAT master backups
+#
+# @example
+#   include profile_xcat::master::backup
+class profile_xcat::master::backup {
+  include ::profile_backup::client
```

Comment:
Some projects might not do backups (for whatever reason).

Should wrap this whole class in an `if $profile_backup::client::enable` condition.